openlr/openlr_stat/openlr_stat.cpp

```diff
@@ -48,6 +48,8 @@ DEFINE_string(ids_path, "", "Path to a file with segment ids to process.");
 DEFINE_string(countries_filename, "",
               "Name of countries file which describes mwm tree. Used to get country specific "
               "routing restrictions.");
+// TODO(mgsergio): Remove this when there is only one algo.
+DEFINE_bool(new_algo, false, "Use new decoding algorithm");
```

Comment:
DEFINE_int32(version, 2, "Version of decoding algorithm")
?

Reply:
optional, of course.

Reply:
-new_algo is more verbose (imho) than -version x when you have only two versions.
Changing to -version also requires validating of a version value.

Reply:
But you don't use ```namespace new```, right? You use ```namespace v2```. So, from my point of view, this is quite misleading. Moreover, you don't plan to remove old decoder, do you?